backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inout/impl/ReceiptInOutLineHUAssignmentListener.java

```diff
@@ -54,6 +58,7 @@
 public final class ReceiptInOutLineHUAssignmentListener extends HUAssignmentListenerAdapter
 {
 	public static final transient ReceiptInOutLineHUAssignmentListener instance = new ReceiptInOutLineHUAssignmentListener();
+	private static final SourceHUsService sourceHuService = SpringContextHolder.instance.getBean(SourceHUsService.class);
```

Comment:
tbh the "static" makes be a bit nervous..

Reply:
yeah, right.. dropping it